index.js

```diff
@@ -10,4 +10,8 @@ https://github.com/fluid-project/first-discovery-server/raw/master/LICENSE.txt
 
 "use strict";
 
+var fluid = fluid || require("infusion");
+var gpii = fluid.registerNamespace("gpii");
 require("./src/js/firstDiscoveryServer.js");
+
+gpii.firstDiscovery.server();
```

Comment:
Basically an expander that calls fluid.stringTemplate and updates the baseUrl value with the port, as seen here:

https://github.com/the-t-in-rtf/gpii-express-user/blob/master/tests/js/test-environment.js